modules/unsupported/cql-json/src/test/java/org/geotools/filter/text/cqljson/CQLJsonTest.java

```diff
@@ -0,0 +1,16 @@
+package org.geotools.filter.text.cqljson;
```

Comment:
Copyright header missing  (can you add it to all test classes?)

Reply:
Added